src/main/java/com/krystian/RockPapersScissorsWinner.java

```diff
@@ -0,0 +1,49 @@
+package com.krystian;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class RockPapersScissorsWinner {
+    private List<String> listOfStrings = new ArrayList<>();
+
+    public int pareAllElementsToInteger(String path) throws IOException {
+        char array[] = {'A', 'B', 'C'};
+        char array2[] = {'X', 'Y', 'Z'};
+        listOfStrings = Files.readAllLines(Paths.get(path));
+        int count = 0;
+        for (int x = 0; x < listOfStrings.size(); x++) {
+            for (int y = 0; y < array.length; y++) {
+                if (listOfStrings.get(x).charAt(0) == array[y]) {
+                    if (y == 0) {
+                        if (listOfStrings.get(x).charAt(2) == 'X')
+                            count = count + 4;
```

Comment:
wszelkie + 4 , +5 itd powinno byc to w zmiennych finalnych. I z nazwa. Bo ja nie mam zielonego pojecia czemu akurat dodajesz 4 a nie 84845